src/uct/sm/mm/base/mm_iface.h

```diff
@@ -49,7 +49,7 @@ struct uct_mm_fifo_ctl {
 
     /* 2nd cacheline */
     volatile uint64_t  tail;       /* how much was read */
-} UCS_S_PACKED;
+} UCS_S_PACKED UCS_V_ALIGNED(UCS_SYS_CACHE_LINE_SIZE);
```

Comment:
why not ```UCT_UD_SKB_ALIGN``` instead of ```UCS_V_ALIGNED(UCS_SYS_CACHE_LINE_SIZE)```?

Reply:
this is MM transport.. also it relies on another macro which does cacheline padding..